packages/devextreme/testing/tests/DevExpress.ui.widgets/chatParts/avatar.tests.js

```diff
@@ -63,5 +66,64 @@ QUnit.module('ChatAvatar', moduleConfig, () => {
                 assert.strictEqual(this.$element.text(), expectedInitials);
             });
         });
+
+        QUnit.test('img element should have correct alt attribute', function(assert) {
```

Comment:
Let's add a separate module like `image rendering` and move these tests there. In the future there will be also tests for `avatarAlt`.

P.s. don't forget to add a task about `avatarAlt` to aggregator